fv3core/stencils/fv_dynamics.py

```diff
@@ -325,6 +330,7 @@ def __init__(
             comm, namelist, self._ak, self._bk, self._phis
         )
         self._hyperdiffusion = HyperdiffusionDamping(self.grid)
+        self._do_cubed_to_latlon = CubedToLatLon(self.grid, True)
```

Comment:
For readability `do_halo_updates=True`

Reply:
We decided to make `do_halo_updates` optional so that the interaction with the global halo flag is more clear.